docs/compiling/compiling-on-debian.md

```
@@ -0,0 +1,84 @@
+
+## Supported OS
+
+- Debian
+
+## 1. Install the required software
+
+The following command will install Git, CMake, a compiler and the libraries used by Canary. 
+
+Git will be used to download the source code, and CMake will be used to generate the build files.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)